src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -129,12 +134,94 @@ define(function (require) {
             map.addControl(new FitControl());
           }
 
+          // add tooltips
+          self._attr.disableTooltips = false;
+
+          if (self._attr.addLeafletPopup && self.tooltipFormatter && !self._attr.disableTooltips) {
```

Comment:
Use the standard `vis.params.addTooltip` instead of `vis.params.addLeafletPopup`